docs/settingup.md

```diff
@@ -1165,6 +1165,86 @@ Unpublish sheets in the current app.
 }
 ```
 
+</details><details>
+<summary>subscribeobjects</summary>
+
+## Subscribeobjects action
+
+Subscribe to any object in the currently active app.
+
+### Settings
+
+* `clear`: Clear any previously subscribed objects from subscription list.
+* `ids`: List of object ID's to subscribe to.
+
+### Example
+
+Subscribe to two objects in the currently active app, unsubscribing to any previous subscriptions. 
+
+```json
+{
+    "action" : "subscribeobjects",
+    "label" : "clear subscriptions and subscribe to mBshXB and f2a50cb3-a7e1-40ac-a015-bc4378773312",
+     "disabled": false,
+    "settings" : {
+        "clear" : true,
+        "ids" : ["mBshXB", "f2a50cb3-a7e1-40ac-a015-bc4378773312"]
+    }
+}
+```
+
+Subscribe to and additional single object (or a list of objects) in the currently active app, adding it to any previous subscriptions. 
```

Comment:
Suggestion:
"Subscribe to an additional single object (or a list of objects) in the currently active app, adding the new subscription to any previous subscriptions. "